arango_rdf/main.py

```diff
@@ -290,52 +295,54 @@ def import_ontology(self, data: str, format: str = "xml") -> None:
             format of the rdf file (default is "xml")
         """
 
-        self.rdf_graph.parse(data, format=format)
-        graph_id = self.rdf_graph.identifier.toPython()
+        raise NotImplementedError("DO NOT USE ontology-specific functions (incomplete)")
```

Comment:
Need an opinion on what to do with this

Noticed the `# DO NOT USE ontology-specific functions, incomplete` comment, so I assumed we don't want users to call `import_ontology()`

Reply:
For now, I think it would be best to remove the ontology code or at least move it off of the main branch